src/components/VDataTable/mixins/head.js

```diff
@@ -1,6 +1,8 @@
 export default {
   methods: {
     genTHead () {
+      if (this.hideHeaders === true) { return } // Exit Early since no headers are needed. Do we also need to set this.headers = [] ?
```

Comment:
Don't ask questions like that in the code, put them in a message in the PR instead.

Reply:
And since `hideHeaders` is bool, `=== true` can be removed

Reply:
*And* you don't need the braces either, all they do is look ugly

Reply:
All noted

Reply:
And to answer the question, you can't set headers, as it is a prop.

Reply:
OK. Got it